src/DotNetCore.CAP.AzureServiceBus/Consumer/ServiceBusConsumerDescriptorBuilder.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) .NET Core Community. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace DotNetCore.CAP.AzureServiceBus.Consumer;
+
+public class ServiceBusConsumerDescriptorBuilder
+{
+    /// <summary>
+    /// Name of the group that the consumer is part of.
+    /// </summary>
+    private string GroupName { get; set; } = null!;
```

Comment:
What about renaming it to SubscriptionName to keep parity with the transport idiom?